### DISCUSSION TITLE
Not a bug, just a question!

### DISCUSSION THREAD

**Discussion by shkeffy** [Vedza/NitroSniperGo#430] [General]
**Describe the question**
I would like to know how to make everything from console print out to my webhook, is there a certain command I can initiate/copy-paste? I'm not familiar with Go-

**Comment by shkeffy** [comment]
@Vedza

**Comment by Vedza** [comment]
Hi, you would need to code a different script in any language that take the output and send it to a webhook. But you can already use the webhook feature of the sniper to send every event

**Comment by shkeffy** [comment]
Oh, I didn't know you could use it to send EVERYTHING!!!
Is there some code that I need to change, or is there an option for it-?

**Reply by Vedza** [reply]
I actually coded that so long ago I can't remember but I think yeah hahaha

**Reply by shkeffy** [reply]
Haha

**Comment by shkeffy** [comment]
I will go look.

**Comment by shkeffy** [comment]
Ah. there is!

**Comment by shkeffy** [comment]
Hm... 'good_only'? I sent a test code and it didn't say anything.

**Reply by Vedza** [reply]
I think if you put good_only to false it should send failed code

**Reply by shkeffy** [reply]
It did not :/

**Reply by shkeffy** [reply]
Also, is there an anti-token taker? I had my token stolen, so I had to change my password (It resets your token), but I was wondering if it's possible for it to not grab them?

**Reply by Vedza** [reply]
You can just look at the code, see there isn't one and compile it yourself and you're good. There isn't such thing as "anti-token taker" possible, there is just a grabber or there is not

**Reply by shkeffy** [reply]
Ah, I see.
